app/controllers/mobile_capture_controller.rb

```diff
@@ -1,5 +1,7 @@
 class MobileCaptureController < ApplicationController
   def new
+    SecureHeaders.append_content_security_policy_directives(request,
+                                                            script_src: ['\'unsafe-eval\''])
```

Comment:
Could we write this as:

```ruby
script_src: %w['unsafe-eval']
```
and avoid the escaped quotes?

Also maybe drop a comment that `unsafe-eval` is specifically for the vendor JS that uses wasm?

Reply:
sure.  good ideas!

Reply:
hmmm linter complaining...any idea or should I revert?

Reply:
Bummer! Maybe switch to double quotes so you don't have to escape the singles?

```ruby
["'unsafe-eval'"]
```

or rubocop might not like it either, but like a weirder quote syntax like `%q|`

Reply:
It's not that big of a deal to me either way, the comment is more important to me than the quote scheme